app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -18,6 +18,19 @@
   padding-left: 24px;
   border-left: 1px solid #F4F4F4;
   border-bottom: 1px solid #F4F4F4;
+
+  @media screen and (max-width: 1120px) {
+    flex-direction: column;
+  }
+
+  @media screen and (max-width: 600px) {
+    padding: 18px 20px;
+    width: 375px;
+  }
+
+  @media screen and (max-width: 568px) {
+    width: 320px;
```

Comment:
```suggestion
```

I think you can actually just get rid of these explicit widths, since you can just let the text flexibly expand to fit the size of the screen. This will allow the page to look nicer across a wider range of screen sizes.